providers/google/src/airflow/providers/google/cloud/log/stackdriver_task_handler.py

```diff
@@ -106,7 +109,7 @@ def __init__(
             warnings.warn(
                 "Param `name` is deprecated and will be removed in a future release. "
                 "Please use `gcp_log_name` instead. ",
-                RemovedInAirflow3Warning,
+                RemovedInFutureProviderWarning,
```

Comment:
Maybe we can just use `AirflowProviderDeprecationWarning` ?

Reply:
Ah, good point, did not know this exist. Let me rework.